File: Mathématiques/4 – Suites numériques/5 - Crapauds et couleuvres.py
nbCrapauds = int(input())
nbCouleuvres = int(input())
nbAnnees = int(input())
for i in range(nbAnnees):
    print(nbCrapauds, nbCouleuvres, sep=" ")
    tempCrapaud = 3*nbCrapauds-2*nbCouleuvres
    nbCouleuvres = nbCrapauds - nbCouleuvres
    nbCrapauds = tempCrapaud

File: Mathématiques/4 – Suites numériques/5 - Crapauds et couleuvres.cpp
#include <iostream>
using namespace std;
int main()
{
  int nbCrapauds, nbCouleuvres, nbAnnees;
  cin >> nbCrapauds >> nbCouleuvres >> nbAnnees;
  for (int i = 0; i < nbAnnees; i++)
  {
    cout << nbCrapauds << " " << nbCouleuvres << endl;
    int tempnbCrapauds = 3 * nbCrapauds - 2 * nbCouleuvres;
    nbCouleuvres = nbCrapauds - nbCouleuvres;
    nbCrapauds = tempnbCrapauds;
  }
}